truffle/com.oracle.truffle.tools.debug.shell/src/com/oracle/truffle/tools/debug/shell/server/REPLServer.java

```diff
@@ -203,6 +209,17 @@ public LocationPrinter getLocationPrinter() {
         return locationPrinter;
     }
 
+    void run() {
+        try {
+            Source source = Source.fromNamedText("", "START");
+            source = source.withMimeType(getLanguages().iterator().next().getMimeTypes().iterator().next());
```

Comment:
```java
Source source = Source.fromNamedText("", "START");
getLanguages().iterator().next().eval(source)
```
might "work" as well. But I write "work" in quotes as the code doesn't even know what language is being initialized! The whole approach is wrong. The API hasn't been designed to support this use-case. Alas, it hasn't be designed to make it impossible either.